.github/containers/cs9.Containerfile

```diff
@@ -0,0 +1,80 @@
+FROM quay.io/centos/centos:stream9 as builder
+
+ARG WAR_FILE
+
+USER root
+
+# Update and install dependencies
+RUN dnf -y --setopt install_weak_deps=False update && \
+    dnf -y --setopt install_weak_deps=False install java-17-openjdk-devel openssl wget unzip && \
+    dnf clean all
+
+# Prepare Tomcat
+COPY apache-tomcat-9.0.76.tar.gz /tmp/
+RUN tar xzf /tmp/apache-tomcat-9.0.76.tar.gz -C /tmp && \
+    mkdir /opt/tomcat && \
+    mv /tmp/apache-tomcat-9.0.76/* /opt/tomcat/
+
+# Prepare Candlepin
+RUN mkdir -p /app/build
+WORKDIR /app/build
+COPY ${WAR_FILE} ./candlepin.war
+
+# Prepare development certs
+RUN mkdir -p /app/certs
+WORKDIR /app/certs
+COPY ./bin/deployment/gen_certs.sh .
+RUN ./gen_certs.sh --cert_out ./candlepin-ca.crt --key_out ./candlepin-ca.key --hostname candlepin; \
+    rm gen_certs.sh;
+
+FROM quay.io/centos/centos:stream9
```

Comment:
@nikosmoum I forget, but were we intentional about wanting to test `ubi9-minimal:latest` base image in our spec tests? Introducing this cs9.Containerfile and using it in our spec tests GH actions means we no longer test against `ubi9-minimal:latest`.

Reply:
I remember the conversation now, and it was around wanting to test both rhel8 and rhel9 equivalent in our spec tests. So, this change is fine!